docs/sections/pipeline_samples/tutorials/clean_existing_dataset.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.        load_dataset &gt;&gt; evaluate_responses &gt;&gt; [keep_columns, to_argilla]
</pre><p>does this work? I would say we first filter out the columns? or do we also do that within the to_argilla Step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Yes, I'll change it for this one</p>